### PR TITLE
SDK Update - Customer Service

### PR DESCRIPTION
**Pull request by imvanzen**
Added following methods:
- Create Customer
- Update Customer
- Fetch Customer
- Delete Customer

All documented, verified

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -591,6 +591,101 @@ Possible errors are:
...
+
+result = voucherify.customer.update(payload)
+```
+
+Result:
+```json
+{
+    "id": "cust_gVYAaioitMz3GO6HSKFLf7Or",
+    "created_at": "2016-06-12T16:03:36Z",
+    "description": "Improved description about Melanie personality",
```
Comment (pawelrychlik) [inline README.md]: why is it different?

File: README.md [hunk 2]
```diff
@@ -591,6 +591,101 @@ Possible errors are:
...
+
+#### Create customer
+
+Example:
+
+```python
+payload = {
+    "name": "Melanie Morgan",
+    "email": "melanie@email.com",
+    "description": "Description of Melanie personality for test reason",
```
Comment (pawelrychlik) [inline README.md]: what does it even mean? :)

File: README.md [hunk 3]
```diff
@@ -591,6 +591,101 @@ Possible errors are:
...
+    "object": "customer"
+}
+```
+
+#### Fetch customer
+
+Example:
+
+```python
+result = voucherify.customer.fetch("cust_gVYAaioitMz3GO6HSKFLf7Or")
```
Comment (pawelrychlik) [inline README.md:632]: I'd rather `get` a customer, not `fetch` it. Why such naming convention?

File: README.md [hunk 4]
```diff
@@ -591,6 +591,101 @@ Possible errors are:
...
+#### Delete customer
+
+Example:
+
+```python
+result = voucherify.customer.delete("cust_gVYAaioitMz3GO6HSKFLf7Or")
+```
+
+Result:
+`This method doesn't return result`
```
Comment (pawelrychlik) [inline README.md]: technically, the result is an empty message body.

btw this is inconsistent with https://github.com/rspective/voucherify-nodejs-sdk#delete-customer

File: README.md (hunk 3 above)
Comment (imvanzen) [inline README.md:632]: I think `fetch` is safer name for method, rather than `get` or `read`

File: examples/customer_test.py [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    client_secret_key="3266b9f8-e246-4f79-bdf0-833929b1380c"
+)
+
+"""
+Create Customer
+"""
+payload = {
+    "name": "Melanie Morgan",
+    "email": "melanie@email.com",
+    "description": "Description of Melanie personality for test reason",
```
Comment (pawelrychlik) [inline examples/customer_test.py]: Oo

**Pull request merged**